notebooks/wb-population/wb-population.py

```
@@ -0,0 +1,151 @@
+# We recommend using `pooch` to download input data. `Pooch` will verify that the
+# downloaded file matches the expected hash as a check that the download was performed
+# successfully.
+
+# %%
+
+# Data has been cached by CR (hash came from original download)
+data_fname = metadata.download_file()
+
+# %%
+zf = zipfile.ZipFile(data_fname)
+zf.filelist
```

Ruff (0.15.21)

[warning] Found useless expression. Either assign it to a variable or remove it.

(B018)